src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/Internal/Json/MessageTypeInfoResolver.cs

```diff
@@ -63,7 +63,7 @@ public MessageTypeInfoResolver(JsonContext context)
 
     private bool IsStandardMessage(Type type, [NotNullWhen(true)] out MessageDescriptor? messageDescriptor)
     {
-        if (!typeof(IMessage).IsAssignableFrom(type))
+        if (type.IsInterface || !typeof(IMessage).IsAssignableFrom(type))
```

Comment:
Do we have test coverage for this change?

Reply:
This change confused me. When I tested gRPC JSON transcoding in the sandbox app, the MessageTypeInfoResolver was called with interfaces by S.T.J. I haven't seen that before. Maybe it is new behavior in .NET 9, but on the other hand it doesn't show up in tests 🤷 

I guess I can add a unit test to verify.